tests/App/config.yml

```
@@ -0,0 +1,75 @@
+framework:
+    secret: secret
+    templating:
+        engines: ['twig']
+    session:
+        handler_id: session.handler.native_file
+        save_path: '%kernel.project_dir%/var/sessions/%kernel.environment%'
```

```suggestion
        save_path: '%app.base_dir%/var/sessions/%kernel.environment%'
```
So sessions are not created in the working directory